Formula/mage.rb

```diff
@@ -0,0 +1,25 @@
+class Mage < Formula
+  desc "Make/rake-like build tool using Go"
+  homepage "https://magefile.org"
+  url "https://github.com/magefile/mage/archive/v2.2.0.tar.gz"
+  sha256 "9a706d8b351bb395d3baca7ee98856a03f46a33ae3d884281c5841a9b8a4a924"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    dst = buildpath/"src/github.com/magefile/mage"
+    src = Dir[buildpath/"*", buildpath/".*"].reject { |f| f.end_with? "." }.uniq
```

Comment:
What are you trying to achieve here? If you're trying to avoid `.brew_home` being dragged in use:
```ruby
dir.install buildpath.children - [buildpath/".brew_home"]
```

Reply:
I'm trying to get a move a complete copy of mage's source, including its dotfiles, in the appropriate directory underneath `GOPATH`.  Thanks for the helper method info, I'll move to that model presently.